UnityAds/UnityAdsInterstitialCustomEvent.m

```diff
@@ -68,14 +68,14 @@ - (void)requestAdWithAdapterInfo:(NSDictionary *)info adMarkup:(NSString *)adMar
         NSError *error = [self createErrorWith:@"Unity Ads adapter failed to request interstitial ad, Unity Ads is not initialized yet. Failing this ad request and calling Unity Ads initialization so it would be available for an upcoming ad request"
                                      andReason:@"Unity Ads is not initialized."
                                  andSuggestion:@""];
-        MPLogAdEvent([MPLogEvent adLoadFailedForAdapter:NSStringFromClass(self.class) error:error], [self getAdNetworkId]);
+        MPLogAdEvent([MPLogEvent adLoadFailedForAdapter:NSStringFromClass(self.class) error:error], placementId);
         [self.delegate fullscreenAdAdapter:self didFailToLoadAdWithError:error];
         
         return;
     }
     
     [UnityAds load:self.placementId loadDelegate:self];
```

Comment:
I think load needs to load `placementId` and not `self.placementId` here.
```
(lldb) po placementId
video

(lldb) po self.placementId
 nil
 ```

Reply:
true it should be .